apps/src/applab/applab.js

```diff
@@ -1351,7 +1374,7 @@ Applab.serializeToLevelHtml = function () {
   var divApplab = document.getElementById('divApplab');
   var clone = divApplab.cloneNode(true);
   // Remove unwanted classes added by jQuery.draggable.
-  $(clone).find('*').removeAttr('class');
+  $(clone).find('*').removeClass('ui-draggable ui-draggable-handle');
```

Comment:
Apologies for leaving this mess for you, but it seems like it might be cleaner to do something like:

      $(clone).children().each(function() {
        if ($(this).data('uiDraggable')) {
          $(this).draggable('destroy');
        }
      });

If that doesn't work for some reason, then your way is still an improvement over what existed before.